### PR TITLE
refactor: Fixed some typos

### PR DESCRIPTION
**Pull request by mikayelgr**
I have fixed some typos in the code's comments.

### CODE REVIEW

**Review by MattTheCuber — APPROVED** [review]

**Review by peepopoggers — APPROVED** [review]

**Review by mmaneetsingh — APPROVED** [review]
Thanks for contributing this change.
Change looks good, we will be taking it in the development branch.

**Comment by michaelgrigoryan25** [comment]
Sure, let me rename it quickly.

File: kernel-open/common/inc/nv_uvm_interface.h [hunk 1]
```diff
@@ -194,7 +194,7 @@ void nvUvmInterfaceAddressSpaceDestroy(uvmGpuAddressSpaceHandle vaSpace);
     nvUvmInterfaceMemoryAllocFB
 
     This function will allocate video memory and provide a mapped Gpu
-    virtual address to this allocation. It also returns the Gpu physical
+    virtual address to this allocation. It also returns the Gpu's physical
```
[the remark this answers is not in the document]
Comment (michaelgrigoryan25) [reply kernel-open/common/inc/nv_uvm_interface.h]: Fixed in 4912e73

File: kernel-open/common/inc/nv_uvm_interface.h [hunk 2]
```diff
@@ -80,7 +80,7 @@ NV_STATUS nvUvmInterfaceRegisterGpu(const NvProcessorUuid *gpuUuid, UvmGpuPlatfo
     nvUvmInterfaceUnregisterGpu
 
     Unregisters the GPU with the provided UUID. This drops the ref count from
-    nvUvmInterfaceRegisterGpu. Once the reference count goes to 0 the device may
+    nvUvmInterfaceRegisterGpu. Once the reference count is equal to 0 the device may
```
[the remark this answers is not in the document]
Comment (michaelgrigoryan25) [reply kernel-open/common/inc/nv_uvm_interface.h]: Fixed in 4912e73

File: kernel-open/common/inc/nv_uvm_interface.h [hunk 3]
```diff
@@ -194,7 +194,7 @@ void nvUvmInterfaceAddressSpaceDestroy(uvmGpuAddressSpaceHandle vaSpace);
     nvUvmInterfaceMemoryAllocFB
 
     This function will allocate video memory and provide a mapped Gpu
-    virtual address to this allocation. It also returns the Gpu physical
+    virtual address to this allocation. It also returns the Gpu physical
```
Comment (yashank09) [inline kernel-open/common/inc/nv_uvm_interface.h:197]: Not sure why this is needed when a new line already exists? :smiley:

Comment (michaelgrigoryan25) [reply kernel-open/common/inc/nv_uvm_interface.h:197]: It is not a new line. It's just a space that I accidentally added. Thanks for pointing it out!

Comment (michaelgrigoryan25) [reply kernel-open/common/inc/nv_uvm_interface.h]: Removed in 298b692.

**Comment by mtijanic** [comment]
Thank you for the PR, but marking as `cosmetic-postponed` for now. Please see https://github.com/NVIDIA/open-gpu-kernel-modules/discussions/190 for details.

**Comment by mtijanic** [comment]
Thank you for the PR, but marking as `cosmetic-postponed` for now. Please see https://github.com/NVIDIA/open-gpu-kernel-modules/discussions/190 for details.

**Pull request closed without merging**